flight/PiOS/STM32F4xx/pios_flash_internal.c

```diff
@@ -34,6 +36,9 @@
 #include "pios_semaphore.h"
 #include <stdbool.h>
 
+/* Todo: consider relocating to a good header */
+bool __attribute__((weak)) vehicle_is_armed = false;
```

Comment:
This is kinda ugly IMO. Would be much better to keep the flash API clean and have the module call into it.